documentation/modules/ROOT/pages/operations/debezium-ui.adoc

```diff
@@ -23,7 +23,35 @@ The UI is implemented as a https://quarkus.io/[Quarkus]-based web application.
 
 == Installation and Configuration
 
-==== Debezium UI container image
+=== Debezium environment vars
+
+[cols="1,2,3,6",options="header"]
+
+|===
+|Item |ENV vars |application.properties vars |description
+
+|1
+|`UI_BASE_URI`
+|ui.base.uri
+|http://localhost:8080/api
```

Comment:
I thought that's not needed (or even present) any longer? The UI should know its own base URI after all, right?

Reply:
It is available in BE https://github.com/debezium/debezium-ui/blob/master/backend/src/main/java/io/debezium/configserver/rest/FrontendConfigResource.java#L21-L38 and some of these are being used here. https://github.com/debezium/debezium-ui/blob/master/oc-demo/05-ui.yaml#L20 but I am not sure about the purpose.

Reply:
Like we discussed earlier, it's only passed to the frontend. When it's not needed there (was previously added to `artifact.url` IIRC) it can be removed. @uidoyen please create a Jira task if it can be removed.